src/modules/lists.js

```diff
@@ -0,0 +1,66 @@
+import Counter from './items.js';
+
+export default class {
+  constructor(menu, list) {
+    this.menu = menu;
+    this.list = list;
+    this.categories = ['seafood', 'pasta', 'vegan'];
+    this.buildMenu();
+  }
+
+  buildMenu = () => {
+    const fragment = new DocumentFragment();
+    this.categories.forEach((category) => {
+      const element = document.createElement('a');
+      element.href = `#${category}`;
+      element.textContent = category;
+      fragment.appendChild(element);
+    });
+    this.menu.appendChild(fragment);
+  }
+
+  displayList = ({ category, listItems, totalLikes }) => {
+    this.likes = totalLikes;
+    const fragment = new DocumentFragment();
+    listItems.forEach((item) => {
+      const element = document.createElement('li');
+      element.id = `item${item.idMeal}`;
+      let html = `<img src="${item.strMealThumb}" alt="${item.strMeal}" />`;
+      html += `<h2>${item.strMeal}</h2>`;
+      html += `<span>${this.countLikes(item.idMeal)} likes</span>`;
+      html += '<button id="likes" type="button">likes</button>';
+      html += '<button id="comments" type="button">comments</button>';
+      html += '<button id="reservations" type="button">reservations</button>';
+      element.innerHTML = html;
```

Comment:
What if you use here `textContent` instead of `innerHtml` ? Because here we're dealing directly with string format of html elements

Reply:
Thanks for the feedback. However, this is a string that contains html needs to be parsed not plain text like the popup data.
You always should use `innerHTML` for parsing html tags inside a string and use `textContent` for plain text without any html tags.